subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/util/DeviceSettingsChecker.kt

```diff
@@ -32,7 +32,7 @@ class DeviceSettingsChecker(private val context: Context) {
     }
 
     private fun checkSupportedApi() {
-        if (Build.VERSION.SDK_INT >= 30) {
+        if (Build.VERSION.SDK_INT > Build.VERSION_CODES.Q) {
```

Comment:
(just grunting) It'll be easier for me to see in Android documentation just numbers instead of these meaningless and hardly comparable names

Reply:
yep, that's really less clear than before
it's magic number rule, and could be suppressed; what do you think?

Reply:
In this case, I'd want to see 30, because it's our logic.

In common Android libraries, version names are valid because they are referenced in the documentation.

up to you